### PR TITLE
[W5.7][W10-1]Tian Xueyan

### PR DESCRIPTION
**Pull request by xueyantian**
Change FindCommand; add new SortCommand.

### CODE REVIEW

**Comment by waytan** [comment]
The implementations seem good, but there are multiple code changes in the single PR (JUnit Tests, case-insensitive FindCommand and new SortCommand).

**Comment by kzzj217** [comment]
Agree with Way Tan. They are good changes and can be separated from this PR and make a new one.

File: test/java/seedu/addressbook/util/TestUtil.java [hunk 1]
```diff
@@ -130,4 +130,19 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
...
+        assertTrue(Utils.isAnyNull("abc", null));
+        assertTrue(Utils.isAnyNull(1, "abc", null));
+        assertTrue(Utils.isAnyNull(new Object(), null, 1, null));
+
+        //the expected output should be false
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(1, 2));
+        assertFalse(Utils.isAnyNull("abc", 1, 'a'));
+        assertFalse(Utils.isAnyNull(new Object()));
+    }
```
Comment (damithc) [inline test/java/seedu/addressbook/util/TestUtil.java:147]: In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch

**Review by damithc — APPROVED** [review]
:+1: Good effort on this PR @xueyantian. In particular, good job on,
* updating docs
* updating I/O tests

Suggestions to improve further:
* Update JUnit tests
* Limit the PR to one enhancement and avoid including changes not related to that enhancement or unmerged  code from other PRs

**Comment by damithc** [comment]
As before, please close the PR after viewing comments.

**Pull request closed without merging**